server/test/app.test.js

```diff
@@ -0,0 +1,29 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+const app = require("../app.js");
```

Comment:
```suggestion
const { server } = require("../app.js");
```
Destructuring the app object should fix your issue